client/client_test.go

```diff
@@ -70,3 +70,30 @@ func TestParseHost(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateClientVersion(t *testing.T) {
+	cases := []struct {
+		v string
+		p string
+		q url.Values
+		e string
+	}{
+		{"1.20", "/containers/json", nil, "/v1.20/containers/json"},
+		{"v1.21", "/containers/json", url.Values{}, "/v1.21/containers/json"},
```

Comment:
Yeah, I just looked at the test above, and assumed it was required/expected behavior.